src/vuestic-theme/vuestic-components/vuestic-grid/_grid-global-styles.scss

```diff
@@ -0,0 +1,256 @@
+// Shamelessly stolen from vuetify.
+// Also see material design material.io/guidelines/layout/responsive-ui.html#responsive-ui-breakpoints
+
+@import "grid-variables";
+// Mixins are reserved for "smart stuffs", flex one-liners are kept intact
+@import "grid-mixins";
+
+.va-layout {
```

Comment:
Rename mixins as well as global styles.

Reply:
i'll do this after turning off bootstrap to prevent conflicts.